src/accountsdb/db.zig

```diff
@@ -256,27 +258,7 @@ pub const AccountsDB = struct {
         var timer = std.time.Timer.start() catch unreachable;
         timer.reset();
 
-        // read the account files
-        var accounts_dir = try std.fs.cwd().openDir(accounts_path, .{ .iterate = true });
-        defer accounts_dir.close();
-
-        const accounts_dir_iter = accounts_dir.iterate();
-
-        var files = try readDirectory(self.allocator, accounts_dir_iter);
-        const filenames = files.filenames;
-        defer {
-            files.filenames.deinit();
-            self.allocator.free(files.filename_memory);
-        }
-
-        var n_account_files: usize = 0;
-        for (filenames.items) |filename| {
-            var fiter = std.mem.tokenizeSequence(u8, filename, ".");
-            const slot = std.fmt.parseInt(Slot, fiter.next().?, 10) catch continue;
-            if (fields.file_map.contains(slot)) {
-                n_account_files += 1;
-            }
-        }
```

Comment:
Why was this code here in the first place? Is it possible that the map could be out of sync with the filesystem?

Reply:
If the file map of the snapshot doesn't accurately describe the files of the snapshot it's in, that'll become apparent later on in `loadAndVerifyAccountsFiles` if any of the files described by the file map don't exist - if any files beyond what it describes exist, we also don't care anyways, since it has to be included in the file map. This check is just unnecessary work, especially when the filesystem could contain hundreds or thousands of files, and the actual file map could realistically contain a mere fraction of that.